### ISSUE TITLE
Ambiguous text for Sig Store X (formerly known as Twitter) link

### ISSUE THREAD

**Issue by krishnabodawala** [sigstore/docs#277]
## Description

Link for the Sig Store X (formerly known as Twitter) page does not have meaningful text, creating ambiguity for a screen reader user.

Recommendation: Link text should be I.E. 'Connect us on X' or 'Sig Store on X' to make it more meaningful.

## Note

The relevant WCAG success criteria is[2.4.4]( https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)
Reference Accessibility Audit #245

## Background

Test performed by Krishna Bodawala Accessibility Consultant Iota

**Comment by smythp** [comment]
Created a PR for this. I just went with "Twitter" and "GitHub" for now., rather than "Connect to Sigstore on Twitter" or similar.
